### DISCUSSION TITLE
When an item is dropped, other sortable items move unexpectedly

### DISCUSSION THREAD

**Discussion by spidertwin2** [clauderic/dnd-kit#867] [Q&A]
Hello,

When an item is dragged to the right, other items move unexpectedly to the left before shifting back into place.  When an item is moved to the left and dropped, it unexpectedly travels a bit more to the left before reaching its position.

Why this sudden left shift when items are dropped?  Please see sandbox [link.](https://codesandbox.io/s/upbeat-burnell-zooglp?file=/src/index.tsx)

Thanks 🌴

**Comment by ivanjeremic** [comment]
I have the same issue.

**Comment by pstrassmann** [comment]
Try replacing your Chakra `<HStack>` component with `<div style={{display: 'flex', gap: '1rem'}}>`. I know this doesn't get to the root of the issue but may help you troubleshoot. I believe this error has something to do with the styling on this parent component.

**Reply by spidertwin2** [reply]
I replaced `<HStack>` with `<div>`, but the problem still persists.

**Reply by pstrassmann** [reply]
See this codesandbox: https://codesandbox.io/s/exciting-mendeleev-o2gupw. I've just replaced the `<HStack>` with the `<div>` as suggested and I'm no longer seeing the shift on drag start or end. Is the problem still persisting there?
